gobblin-core/build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
   compile externalDependency.findBugsAnnotations
   compile externalDependency.oltu
   compile externalDependency.opencsv
+  compile externalDependency.pegasus.d2
```

Comment:
Let's just create a separate module for this. We need to try not to add unnecessary dependencies to `gobblin-core` as we move forward.